support/@trustfractal/data-host/package.json

```diff
@@ -1,25 +1,24 @@
 {
   "name": "@trustfractal/data-host",
```

Comment:
I'd like to have the Data Host library in /data_host/library or something similar. I think /support should really only be for very minimal focused things that we use to support our own development.